server/services/participants-helper.js

```diff
@@ -183,15 +183,28 @@ class FieldsFilteredParticipantsFinder {
 
     if (user.isEmployer || user.isHA) {
       this.context.table = this.context.table.join({
-        [employerSpecificJoin]: {
-          type: 'LEFT OUTER',
-          relation: collections.PARTICIPANTS_STATUS,
-          on: {
-            participant_id: 'id',
-            current: true,
-            employer_id: user.id,
-          },
-        },
+        ...(statusFilters.includes('hired')
+          ? {
+              [employerSpecificJoin]: {
+                type: 'LEFT OUTER',
+                relation: collections.PARTICIPANTS_STATUS,
+                on: {
+                  participant_id: 'id',
+                  current: true,
+                },
+              },
+            }
+          : {
+              [employerSpecificJoin]: {
+                type: 'LEFT OUTER',
+                relation: collections.PARTICIPANTS_STATUS,
+                on: {
+                  participant_id: 'id',
+                  current: true,
+                  employer_id: user.id,
```

Comment:
Is this correct that employer_id on-clause is for the status filter without 'hired' condition?